### PR TITLE
[content/en/opentelemetry] Mention different signals request different batch processor configs

### PR DESCRIPTION
**Pull request by songy23**
### What does this PR do?

Remind the readers that they may need different batch sizes for batch processor for different signal types (traces/metrics/logs) when using datadog exporter.

### Motivation

The previous comment and config in the batch processor only refers to the payload size limit of trace intake. When customers use the same config for metrics or logs pipeline, they may get 413 errors because metrics and logs have a smaller payload size limit than that in trace. This led to customer confusions and issues being filed in https://github.com/open-telemetry/opentelemetry-collector-contrib.

### Additional Notes

Related customer issues:
- https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16834
- https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17566

Also note that the payload size limits in metrics and logs intake are in the public document, but traces intake doesn't have a public reference.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/opentelemetry/otel_collector_datadog_exporter.md [hunk 1]
```diff
@@ -97,7 +95,10 @@ service:
 
 Where `<DD_SITE>` is your site, {{< region-param key="dd_site" code="true" >}}.
 
-The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries over HTTP and gRPC, and sets up a [batch processor][5], which is mandatory for any non-development environment.
+The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries over HTTP and gRPC, and sets up a [batch processor][5], which is mandatory for any non-development environment. Note that you may get 413 - Request Entity Too Large errors if you batch too much telemetry data in the batch processor. The exact config of batch processor depends on your specific workload as well as the signal types - Datadog intake has different payload size limits for the 3 signal types:
```
Comment (urseberry) [inline content/en/opentelemetry/otel_collector_datadog_exporter.md:98]:
```suggestion
The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries over HTTP and gRPC, and sets up a [batch processor][5], which is mandatory for any non-development environment. Note that you may get `413 - Request Entity Too Large` errors if you batch too much telemetry data in the batch processor.

The exact configuration of the batch processor depends on your specific workload as well as the signal types. Datadog intake has different payload size limits for the 3 signal types:
```

File: content/en/opentelemetry/otel_collector_datadog_exporter.md [hunk 2]
```diff
@@ -97,7 +95,10 @@ service:
...
+- Trace intake: 3.2MB
+- Log intake: https://docs.datadoghq.com/api/latest/logs/
+- Metrics V2 intake: https://docs.datadoghq.com/api/latest/metrics/#submit-metrics
```
Comment (urseberry) [inline content/en/opentelemetry/otel_collector_datadog_exporter.md:101]: Can you turn the bare URLs into links?

Comment (songy23) [reply content/en/opentelemetry/otel_collector_datadog_exporter.md]: Sounds good, done.

**Review by urseberry — APPROVED** [review]

**Comment by songy23** [comment]
Thanks for the review!

**Pull request merged**